WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/ActivityLogListFragment.kt

```diff
@@ -138,7 +139,7 @@ class ActivityLogListFragment : Fragment() {
         swipeToRefreshHelper.isRefreshing = eventListStatus == FETCHING
         // We want to show the progress bar at the bottom while loading more but not for initial fetch
         val showLoadMore = eventListStatus == LOADING_MORE
-        log_list_progress.visibility = if (showLoadMore) View.VISIBLE else View.GONE
+        progress?.visibility = if (showLoadMore) View.VISIBLE else View.GONE
```

Comment:
we can remove this and leave the `progress` always visible

Reply:
This was resolved with [this comment] (https://github.com/wordpress-mobile/WordPress-Android/pull/8372#issuecomment-424982372)